src/marqo/tensor_search/api.py

```diff
@@ -236,7 +236,7 @@ def delete_index(index_name: str, marqo_config: config.Config = Depends(generate
 @app.post("/indexes/{index_name}/documents/delete-batch")
 def delete_docs(index_name: str, documentIds: List[str], refresh: bool = True,
                       marqo_config: config.Config = Depends(generate_config)):
-    return tensor_search.delete_documents(
```

Comment:
Why the import change? (IIRC tensor_search.<OPERATION> is standard).

Reply:
done

Reply:
(as in made it consistent)